get_goes_x.py

```diff
@@ -31,44 +41,63 @@
     print(('Warning: failed to open URL {}: {}'.format(url, last_err)))
     sys.exit(0)
 
-
-colnames = 'year month dom  hhmm  mjd secs short long ratio'.split()
-data_colnames = colnames[-3:]
-
 try:
-    dat = ascii.read(urldat, guess=False, Reader=ascii.NoHeader,
-                     data_start=3, names=colnames)
+    dat = Table(json.loads(urldat))
```

Comment:
Actually, my mistake, I thought the GOES_X warning was from the h5 code, but it looks like it is our own warning on a failure here.  So I'm guessing that 

```
** ERROR - line 5: <<2020-Apr-01 13:13>> Warning: malformed GOES_X data so table read failed: Expecting ',' delimiter: line 1 column 940921 (char 940920)
** ERROR - line 5: <<2020-Apr-01 13:13>> Warning: malformed GOES_X data so table read failed: Expecting ',' delimiter: line 1 column 940921 (char 940920)
** ERROR - line 10: <<2020-Apr-01 21:48>> Warning: malformed GOES_X data so table read failed: Expecting ',' delimiter: line 1 column 941067 (char 941066)
** ERROR - line 10: <<2020-Apr-01 21:48>> Warning: malformed GOES_X data so table read failed: Expecting ',' delimiter: line 1 column 941067 (char 941066)
```

is either a hiccup with the fetch or this is a race condition on the swpc webserver and we're occasionally fetching an incomplete or empty file.  I'm wondering about either printing out the data to check and also chucking the warning in the logs because I don't think we care.

Reply:
Maybe it would be worth putting this into the log, but without ERROR or warning.  This way we get some visibility into how often this is happening without spamming us with ignorable warnings.